cores/portduino/AsyncUDP.cpp

```diff
@@ -19,6 +19,10 @@ AsyncUDP::~AsyncUDP() {
     uv_async_send(&_async);
     _ioThread.join();
     uv_loop_close(&_loop);
+    if (_fd > 0) {
+        close(_fd);
```

Comment:
The libuv docs are not clear to me on who takes ownership of closing the fd when using `uv_udp_open`.